src/Compilers/Test/Utilities/VisualBasic/CompilationTestUtils.vb

```diff
@@ -254,6 +263,28 @@ Friend Module CompilationUtils
         Return c
     End Function
 
+#If TEST_OPERATION Then
+    Private Sub WalkOperationTrees(compilation As VisualBasicCompilation)
+        Dim operationWalker = TestOperationWalker.GetInstance()
+
+        For Each tree In compilation.SyntaxTrees
+            Dim semanticModel = compilation.GetSemanticModel(tree)
+            Dim root = tree.GetRoot()
+
+            ' need to check other operation root as well (property, etc)
+            For Each methodNode As MethodBlockSyntax In root.DescendantNodesAndSelf().Where(Function(n)
+                                                                                                Dim kind = n.Kind()
+                                                                                                Return kind = SyntaxKind.FunctionBlock OrElse kind = SyntaxKind.SubBlock
+                                                                                            End Function)
+                For Each statement In methodNode.Statements
```

Comment:
Why do you need to loop through each individual statement? Statements is a StatementSyntax.